tests/integration/utils.py

```diff
@@ -93,6 +94,34 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, request_json={}):
+        """
+        Fetch a list of requests from the Cachito API.
+
+        :return: Object that contains response from the Cachito API
+        :rtype: list
+        """
```

Comment:
You should avoid using mutable defaults for keyword arguments. This is because the same object is always used if you call the function multiple times without passing in the keyword argument. For example:
```
>>> import random
>>> def example(param={}):
...     param[random.randint(0, 1000)] = 'value'
...     print(param)
... 
>>> example()
{250: 'value'}
>>> example()
{250: 'value', 838: 'value'}
>>> example()
{250: 'value', 838: 'value', 545: 'value'}
```

It's not particular harmful in this case since you aren't updating the value, but it's just best to avoid doing this to prevent issues in the future if the scope of the function changes.
```suggestion
    def fetch_all_requests(self, request_json=None):
        """
        Fetch a list of requests from the Cachito API.

        :return: Object that contains response from the Cachito API
        :rtype: list
        """
        if not request_json:
            request_json = {}
```

Reply:
Done, thank you!